tests/test_waypoints_names_to_coordinates.py

```diff
@@ -0,0 +1,66 @@
+"""
+Test process.
+"""
+from modules import waypoint_names_to_coordinates
+
+NAMES_VALID = ["Waterloo", "Aerial", "Robotics", "Group 15"]
+WAYPOINT_DICTIONARY = {
+    "Aerial": (9, 7),
+    "Group 15": (3, 4),
+    "Robotics": (-1, 0),
+    "University of Waterloo Station for 301 ION": (6, 6),
+    "WARG": (8, 2),
+    "Waterloo": (2, -5),
```

Comment:
Nit: Could we make this Waterloo, Aerial, Robotics, Group 15, and then the invalid ones? Instead of Group 15 could we change it to Group

Reply:
"Waterloo": (2, -5),
    "Aerial": (9, 7),
    "Robotics": (-1, 0),
    "Group": (3, 4),
    "University of Waterloo Station for 301 ION": (6, 6),
    "WARG": (8, 2),
    
    Do you want it like this?

Reply:
No, keep the dictionary as is.